python/cudf/dataframe/series.py

```diff
@@ -188,6 +188,8 @@ def __len__(self):
         return len(self._column)
 
     def __getitem__(self, arg):
+        if isinstance(arg, (list,)):
+            arg = Series(arg)
```

Comment:
I think you need to make this recursive otherwise this doesn't return anywhere.

Reply:
I'm not sure what you mean. I cast the list to a Series here because if it is a native list, the tests fail. :) This is one of two places where I chose an efficient developer solution possibly at a greater runtime performance cost - I don't know if `arg = Series(arg)` will be computationally expensive.

I think the alternative is to modify the below code to accept a `list` instance for indexing.

I'm going to double check this and follow up in a few.

Reply:
Woops, misread the if as an elif which is why I made the recursive comment 👍.